types/fixed-data-table-2/index.d.ts

```diff
@@ -0,0 +1,515 @@
+// Type definitions for fixed-data-table-2 0.8
+// Project: https://github.com/facebook/fixed-data-table-2
```

Comment:
Project should make reference to https://github.com/schrodinger/fixed-data-table-2